dotCMS/src/main/java/com/dotmarketing/servlets/BinaryExporterServlet.java

```diff
@@ -576,7 +578,13 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) throws Servl
 			if(!resp.isCommitted()){
 			  resp.sendError(HttpServletResponse.SC_NOT_FOUND);
 			}
-		}catch (Exception e) {
+		} catch (IOException e) {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Combine this catch with the one at line 575, which has the same body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2147)